windows.sh

```diff
@@ -11,14 +11,13 @@
 
 ANSIBLE_PLAYBOOK=$1
 PLAYBOOK_DIR=${ANSIBLE_PLAYBOOK%/*}
-ROLE_REQUIREMENTS=$(find /vagrant/$PLAYBOOK_DIR -name "requirements.yml" -o -name "requirements.txt")
```

Comment:
It's not gone. I moved it to line 53 where it is executed and now also works if multiple requirement files are present.